code/game/objects/effects/decals/remains.dm

```diff
@@ -1,23 +1,39 @@
+/obj/effect/decal/remains
+	gender = PLURAL
+
 /obj/effect/decal/remains/human
 	name = "remains"
 	desc = "They look like human remains. They have a strange aura about them."
-	gender = PLURAL
 	icon = 'icons/effects/blood.dmi'
 	icon_state = "remains"
 	anchored = 1
 
 /obj/effect/decal/remains/xeno
 	name = "remains"
 	desc = "They look like the remains of something... alien. They have a strange aura about them."
-	gender = PLURAL
 	icon = 'icons/effects/blood.dmi'
 	icon_state = "remainsxeno"
 	anchored = 1
 
 /obj/effect/decal/remains/robot
 	name = "remains"
 	desc = "They look like the remains of something mechanical. They have a strange aura about them."
-	gender = PLURAL
 	icon = 'icons/mob/robots.dmi'
 	icon_state = "remainsrobot"
-	anchored = 1
\ No newline at end of file
+	anchored = 1
+
+/obj/effect/decal/remains/slime
+	name = "You shouldn't see this"

```

Comment:
This should never exist in world, since it qdels pretty much immediately. If it exists, something broke.